.github/workflows/tests.yml

```diff
@@ -25,6 +25,8 @@ jobs:
       - uses: actions/checkout@v2
       - name: Tests
         run: cargo test --verbose
+      - name: KAT
```

Comment:
Why "KAT" as opposed to "Test Vectors"? Is there a difference?

Reply:
This is a known answer test, hence the name. The test vector generation should be moved out of this task at some point when everything is more stable.